AwsEncryptionSDK/runtimes/rust/examples/example_utils/utils.rs

```diff
@@ -36,3 +36,238 @@ pub const TEST_LOGICAL_KEY_STORE_NAME: &str = "KeyStoreDdbTable";
 
 pub const TEST_KEY_STORE_KMS_KEY_ID: &str =
     "arn:aws:kms:us-west-2:370957321024:key/9d989aa2-2f9c-438c-a745-cc57d3ad0126";
+
+// ECDH Utils
+use aws_esdk::aws_cryptography_primitives::types::EcdhCurveSpec;
+use std::path::Path;
+use std::io::Write;
+
+pub const TEST_KMS_ECDH_KEY_ID_P256_SENDER: &str =
+    "arn:aws:kms:us-west-2:370957321024:key/eabdf483-6be2-4d2d-8ee4-8c2583d416e9";
+pub const TEST_KMS_ECDH_KEY_ID_P256_RECIPIENT: &str =
+    "arn:aws:kms:us-west-2:370957321024:key/0265c8e9-5b6a-4055-8f70-63719e09fda5";
+
+pub const EXAMPLE_ECC_PRIVATE_KEY_FILENAME_SENDER: &str = "RawEcdhKeyringExamplePrivateKeySender.pem";
+pub const EXAMPLE_ECC_PRIVATE_KEY_FILENAME_RECIPIENT: &str = "RawEcdhKeyringExamplePrivateKeyRecipient.pem";
+pub const EXAMPLE_ECC_PUBLIC_KEY_FILENAME_RECIPIENT: &str = "RawEcdhKeyringExamplePublicKeyRecipient.pem";
+pub const EXAMPLE_KMS_ECC_PUBLIC_KEY_FILENAME_SENDER: &str = "KmsEccKeyringExamplePublicKeySender.pem";
+pub const EXAMPLE_KMS_ECC_PUBLIC_KEY_FILENAME_RECIPIENT: &str = "KmsEccKeyringExamplePublicKeyRecipient.pem";
+
+// Following are the helper functions for running ECDH examples
+
+pub(crate) fn x962_to_x509(
```

Comment:
Yes but that is an extern and it takes in an internal dafny type `ECDHCurveSpec` which I don't want to expose to customers.

Reply:
I would much rather us use the extern code because if the extern were to change then we would have to come and update this as well

Reply:
Once aws-lc-rs supports the various formats we need, we want to drop the dependence on aws-lc-sys.

If we keep the function public, we will have to support it forever. This will make the above update a blocking change. For Rust ECDH examples, I will create a function that mimics X962_to_X509 so that we can update it with the aws-lc-rs conversions once they are supported.